web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -314,8 +314,9 @@
 />
 
 {#if assetInteraction.selectionActive}
-  <div class="sticky top-0">
+  <div class="sticky top-0 z-1">
```

Comment:
What does this fix specifically?

Reply:
In memory viewer, when you select the items, it nav bar is overlayed with the center piece, so this is to fix that

<img width="1651" alt="image" src="https://github.com/user-attachments/assets/c14a2ff3-f8d2-45dc-bf74-421bc58c6939" />